src/vs/workbench/contrib/terminal/browser/terminalTypeAheadAddon.ts

```diff
@@ -510,6 +517,10 @@ class LinewrapPrediction extends NewlinePrediction implements IPrediction {
 		return ' \r';
 	}
 
+	public rollback(cursor: Cursor) {
+		return this.prevPosition ? cursor.moveTo(this.prevPosition) : '';
+	}
+
```

Comment:
this appears to be the same impl as `NewLinePrediction`'s that this extends. Do you need this code?

Reply:
yep you're right